### PR TITLE
ibmcloud fix node_restart

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**
Fixes: #7281

### CODE REVIEW

**Review by petr-balogh — APPROVED** [review]

File: ocs_ci/utility/ibmcloud.py [hunk 1]
```diff
@@ -265,15 +265,12 @@ def restart_nodes(self, nodes, timeout=900, wait=True):
...
-        out = run_ibmcloud_cmd(cmd)
-        worker_nodes = json.loads(out)
-
-        if len(worker_nodes) > 0:
-            for node in worker_nodes:
-                cmd = f"ibmcloud ks worker reboot --cluster {cluster_id} --worker {node['id']} -f"
-                out = run_ibmcloud_cmd(cmd)
-                logger.info(f"Node restart command output: {out}")
+        for node in nodes:
+            provider_id = node.get()["spec"]["providerID"]
```
Comment (jilju) [inline ocs_ci/utility/ibmcloud.py:269]: Is this same as the provider_id which is already assigned in line 265 ?

**Comment by jilju** [comment]
Isn't this a fix for https://github.com/red-hat-storage/ocs-ci/issues/7281 ?

**Comment by gayathrimenath-ibm** [comment]
> Isn't this a fix for #7281 ?

yes

**Comment by petr-balogh** [comment]
Triggered verification job here:
 https://url.corp.redhat.com/02f548c

**Review by vavuthu — DISMISSED** [review]

**Comment by petr-balogh** [comment]
Verified here:
https://url.corp.redhat.com/98d9b62

File: ocs_ci/utility/ibmcloud.py (hunk 1 above)
Comment (vavuthu) [reply ocs_ci/utility/ibmcloud.py:269]: @gayathrimenath-ibm Could you please check this comment.

Comment (petr-balogh) [reply ocs_ci/utility/ibmcloud.py:269]: @gayathrimenath-ibm can you fix this please?

**Review by petr-balogh — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**